packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -237,18 +329,24 @@ public void onOpened(@NonNull CameraDevice device) {
 
           @Override
           public void onClosed(@NonNull CameraDevice camera) {
+            Log.i(TAG, "open | onClosed");
```

Comment:
There's quite a bit of logging in this plugin; is this standard practice for Android code?

Reply:
It's similar to other platforms. It could be useful for debugging of a crash in prod, understanding the sequence of events, etc...

Reply:
it's a judgment call though